Sheet.md

```diff
@@ -0,0 +1,74 @@
+﻿# Sheets
+
+## BottomSheets
+
+### ExpandingBottomSheets
+
+Controls Name: ExpandingBottomSheet
+Control Style Name: MaterialModalStandardBottomSheetStyle
+Properties:
+- IsExpanded: Boolean
+- IsMinimalSheet: Boolean
+- CollapsedContent: Content 
+- CollapsedContentTemplate: DataTemplate
+- MinimalCollapsedContentTemplate: DataTemplate
+- ExpandedCollapsedContent: Content 
+- ExpandedContentTemplate: DataTemplate
+- ExpandedHeaderContentTemplate: DataTemplate
+
+Usage: Place Control in largest area of the current Page/Window to produce optimal behavior.
+
+[see material.io documentation](https://material.io/components/sheets-bottom#expanding-bottom-sheet)
+
+### StandardBottomSheets
+
+Controls Name: StandardBottomSheet
+Control Style Name: MaterialModalStandardBottomSheetStyle
+Properties:
+- Content: Content 
+- ContentTemplate: DataTemplate
+- HeaderContent: Content 
+- HeaderContentTemplate: DataTemplate
+- FullScreenHeaderContentTemplate: DataTemplate
+- SnapAreas: SheetSnapAreaCollection 
+- CurrentSnapArea: SheetSnapArea
+- InitialSnapAreaName: String
+- SheetPosition: Double
+
+Usage: Place Control in largest area of the current Page/Window to produce expected behavior.
+
+[see material.io documentation](https://material.io/components/sheets-bottom#standard-bottom-sheet)
+
+### ModalBottomSheets
+
+Controls Name: ModalBottomSheet
+Control Style Name: MaterialModalStandardBottomSheetStyle
+Properties:
+- IsOpened: Boolean
+- Content: Content 
+- ContentTemplate: DataTemplate
+
+Usage: Place Control in largest area of the current Page/Window, and bind IsOpened to a button.
+
+[see material.io documentation](https://material.io/components/sheets-bottom#modal-bottom-sheet)
+
+### ModalStandardBottomSheets
+A combination between the standard and modal bottom sheets.
+
+Controls Name: ModalStandardBottomSheet
+Control Style Name: MaterialModalStandardBottomSheetStyle
+Properties:
```

Comment:
If you have time to add a summary to each property it would be good!

i.e. SheetPosition: Double // Value between 0.0 and 1.0.  0.0 when sheet is closed and 1.0 when sheet is taking the whole screen.